### DISCUSSION TITLE
Why no score for a single image phishing?

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#3985] [Q&A]
Friends,

I've seen Rspamd score in several spams with more image than text ratio.

In this case, however, check it out, a hacked Wordpress hosting an image. A phishing.

It scored very low and no score due to the image.

Ideas if it's something that can be improved or some setup on our side?

Thanks.
[Fwd- Atendimento ItauDigital EURÍPEDES JUNQUEIRA JR_ ID- 416590.zip](https://github.com/rspamd/rspamd/files/7570331/Fwd-.Atendimento.ItauDigital.EURIPEDES.JUNQUEIRA.JR_.ID-.416590.zip)

**Comment by flaviozluca** [comment]
1 more example.
[one more - ✅  Sr(a), lucaadela@lucanet.com.br Processo Criminal -ID-3279 - Receita Federal (root@b16.info-dfp.org) - 2021-11-19 1202.zip](https://github.com/rspamd/rspamd/files/7572604/one.more.-.Sr.a.lucaadela%40lucanet.com.br.Processo.Criminal.-ID-3279.-.Receita.Federal.root%40b16.info-dfp.org.-.2021-11-19.1202.zip)

**Comment by flaviozluca** [comment]
1 more phishing. this one has an image inside wp-includes. BTW, 
[Bloqueio de Cartao FREE-SX  - [ id 352646812  ] - Atendimento (atendimento@email.com) - 2021-11-19 1116.zip](https://github.com/rspamd/rspamd/files/7572646/Bloqueio.de.Cartao.FREE-SX.-.id.352646812.-.Atendimento.atendimento%40email.com.-.2021-11-19.1116.zip)
wasnt there a rule to score URLs with wp-include?

**Comment by flaviozluca** [comment]
1 more
[example3.zip](https://github.com/rspamd/rspamd/files/7597793/example3.zip)

**Comment by flaviozluca** [comment]
this one not a single image, but several external images. spam. false negative
[several images.zip](https://github.com/rspamd/rspamd/files/7857122/several.images.zip)
.

**Comment by flaviozluca** [comment]
1 more, it scoress 
[App Building - 'Anjali' (anjali.seomanager97@aol.com) - 2022-01-24 1231.zip](https://github.com/rspamd/rspamd/files/8073114/App.Building.-.Anjali.anjali.seomanager97%40aol.com.-.2022-01-24.1231.zip)
empty image a bit... see, spammer using image to bypass scores... cant anything be done to score this case more?

thanks.

**Comment by flaviozluca** [comment]
1 more FN
[imagespam.zip](https://github.com/rspamd/rspamd/files/8111510/imagespam.zip)

**Comment by flaviozluca** [comment]
1 more
[Invoice [G-57902345098 Re-Order Completed. - Ruth Kimberlly (rkimberlly007@gmail.com) - 2022-02-24 1129.zip](https://github.com/rspamd/rspamd/files/8143086/Invoice.G-57902345098.Re-Order.Completed.-.Ruth.Kimberlly.rkimberlly007%40gmail.com.-.2022-02-24.1129.zip)

can someone pls comment?

**Comment by flaviozluca** [comment]
1 more, no score
[spam-image.zip](https://github.com/rspamd/rspamd/files/8217563/spam-image.zip)

**Comment by flaviozluca** [comment]
[Objeto Moveis Para Escritório - 'Kenio - Objeto Moveis - Gmail' (kenioobjetomoveis@gmail.com) - 2022-03-07 1452.zip](https://github.com/rspamd/rspamd/files/8253412/Objeto.Moveis.Para.Escritorio.-.Kenio.-.Objeto.Moveis.-.Gmail.kenioobjetomoveis%40gmail.com.-.2022-03-07.1452.zip)

**Comment by flaviozluca** [comment]
1 more, this one scores blank image a bit
[Hi - Roberty Jamesy (jamesyroberty3@gmail.com) - 2022-03-21 1029.zip](https://github.com/rspamd/rspamd/files/8318832/Hi.-.Roberty.Jamesy.jamesyroberty3%40gmail.com.-.2022-03-21.1029.zip)

**Reply by vstakhov** [reply]
It works merely if an image is within a link. Otherwise, this rule will match innocent attachments like a photo.

**Comment by flaviozluca** [comment]
Hi, Vstakhob

Thanks for your reply!

I see, but still, see how many false negatives? Spams that have a single image. I remember in the Spamassassin days, there was a test that would check the percentage of image in an email, can't that be done?

Also, this seems related. https://github.com/rspamd/rspamd/issues/525

Thanks.

**Reply by vstakhov** [reply]
There is such a check, but it is not necessarily a sign of spam - that's the problem :(

**Reply by flaviozluca** [reply]
Thank for your reply, Vsevolod. I understand, but see how many false negatives? Can't we score a bit more, at least? Proportion of image to text is an important factor, IMHO.

Besides, in some of these samples, I believe image didn't score or scored very very low.

And still, many are phishing.

Appreciate it!

**Comment by flaviozluca** [comment]
1 more
[PROGRAMAÇÃO MAIO 2022 - 'Caminharte Turismo' (caminharte@caminharte.com.br) - 2022-04-21 1325.zip](https://github.com/rspamd/rspamd/files/8743178/PROGRAMACAO.MAIO.2022.-.Caminharte.Turismo.caminharte%40caminharte.com.br.-.2022-04-21.1325.zip)

**Comment by flaviozluca** [comment]
this is a spam, false negative. r_empty_image scored, but too little.
[Transaction ID  #6435R54T65Y76U87 - Nedn Lightp (nednlightp443@gmail.com) - 2022-06-27 1119.zip](https://github.com/rspamd/rspamd/files/8995121/Transaction.ID.6435R54T65Y76U87.-.Nedn.Lightp.nednlightp443%40gmail.com.-.2022-06-27.1119.zip)

**Comment by flaviozluca** [comment]
Major phishing, false negative.
Mainly image and it scored barely anything (the image).
[caixa-phish.zip](https://github.com/rspamd/rspamd/files/9056634/caixa-phish.zip)
